src/WebJobs.Script.WebHost/Resources/Functions/WarmUp/run.csx

```diff
@@ -1,9 +1,26 @@
-﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
IActionResult comes from this namespace. Did this compile without this using?

Reply:
Didn’t remove it. It’s just few lines below :-)

Reply:
Sorry, my diff was completely messed up. That confused me